contact.html

```diff
@@ -0,0 +1,159 @@
+                  
+                </div>
+                </div>
+                <div class="row justify-content-center">
+                    <div class="col-md-3  mb-2 ml-3">
+                        <input type="text" class="form-control" placeholder="Name" id="name">
+                    </div>
+                    <div class="col-md-3 mb-2">
+                        <input type="email" class="form-control" placeholder="Email" id="email">
+                    </div>
+                    <div class="col-md-6 mb-2 mx-3">
+                        <input type="text" class="form-control" placeholder="Subject" id="subject">
+                    </div>
+                    <div class="col-md-6  mb-2 mx-3">
+                        <textarea class="form-control" id="message-area" rows="3" placeholder="Message"></textarea>
+                    </div>
+                    <div class="col-md-6 mb-2 mx-3">
+                        <button class="send-join my-2 w-100">Send</button>
+                    </div>
+                    
+                </div>
+                   <div class="row justify-content-center">
+                    <div class="col-md-6 text-center mt-5">
+                        <h6 class=" fst-italic fw-lighter ">San Francisco, CA 94158</h6>
+                        <h6 class=" fst-italic fw-lighter ">Email: info@mysite.com / Tel: 123-456-7890</h6>
+                    </div>
+                </div>
+                <div class="row ">
+                    <div class=" text-center mt-3 d-flex justify-content-center ">
+                       
+                          <img class="social-icon " src="./img/twitter-icon.svg" alt="twitter-icon">
+                      
+                          <img class="social-icon " src="./img/facebook-icon.svg" alt="facebook-icon">
+                   
+                    </div>
+                </div>
+                <div class="row justify-content-center mt-5 mb-5 ">
+                    <div class="text-center mb-3">
+                        <h6 class=" fst-italic fw-lighter ">Subscribe to my newsletter</h6>
+                    </div>
+                    <div class="col-sm-3 mb-2 ">
+                        <input type="email" class="form-control" placeholder="Enter your email here*" id="email">
+                    </div>
+                    <div class="col-sm-1 ">
+                        <button class="send-join w-100 p-2 mb-5">Join</button>
+                    </div>
+                </div> 
+          
+        
+          <div class="container-fluid  d-flex justify-content-center">
+              <div class="row mt-5">
+            <div class="col-12 map border-0">
+           <iframe src="https://www.google.com/maps/embed/v1/place?key=AIzaSyA0s1a7phLN0iaD6-UE7m4qP-z21pH0eSc&q=Eiffel+Tower+Paris+France" width="1600" height="450"  ></iframe>
```

Comment:
Really nice touch 👍 
Make sure you set `max-width: 100%;` on this though as it's breaking the page at the moment.